### PR TITLE
Gif added

### PR DESCRIPTION
**Pull request by nitish-awasthi**
added_gif

Added Gif for enhancement:

### CODE REVIEW

**Comment by AJV009** [comment]
@nitish-awasthi could you add some description to the PR?

**Comment by nitish-awasthi** [comment]
Description Updated! @AJV009

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Awesome DevC Contributors
+## Awesome DevC Contributors
```
Comment (AJV009) [inline README.md:1]: Hey let the single '#' remain na?
As that is the title of the repo not a subtitle!

Comment (AJV009) [reply README.md:1]: its just a quick small change so that I can merge it directly!

**Comment by nitish-awasthi** [comment]
Rectified! Please have a look @AJV009

**Comment by AJV009** [comment]
> 
> 
> Rectified! Please have a look @AJV009

Sure Boss!
Merged!

**Comment by AJV009** [comment]
Thats a cute little GIF, I like it!
😀😁😄😃😅

**Pull request merged**